File: Lista2/Exercícios_python/Sala/8.py
a = int(input("Digite o valor do número inteiro A: "))
b = int(input("Digite o valor do número inteiro B: "))
c = int(input("Digite o valor do número inteiro C: "))

if a%2==0 and a%3==0:
    if b%2==0 and b%3==0:
        if c%2==0 and c%3==0:
            print("Os números ", a, ",", b, "e", c, "são divisíveis por 2 e por 3")
        else:
            print("Os números ", a, "e", b, "são divisíveis por 2 e por 3")
    else:
        if c%2==0 and c%3==0:
            print("Os números ", a, "e", c, "são divisíveis por 2 e por 3")
        else:
            print("O número ", a, "é divisível por 2 e por 3")
elif b%2==0 and b%3==0:
    if c%2==0 and c%3==0:
        print("Os números ", b, "e", c, "são divisíveis por 2 e por 3")
    else:
        print("O número ", b, "é divisível por 2 e por 3")
else:
    if c%2==0 and c%3==0:
        print("O número", c, "é divisível por 2 e por 3")
    else:
        print("Nenhum dos números inseridos é divisível por 2 e 3")

File: Lista2/Exercícios_C/Sala/8.c
#include <stdio.h>
int main ()

{
    int a, b, c;
    printf("Digite o valor do número inteiro A: ");
    scanf("%i", &a);
    printf("Digite o valor do número inteiro B: ");
    scanf("%i", &b);
    printf("Digite o valor do número inteiro C: ");
    scanf("%i", &c);

    if (a%2==0 && a%3==0) {
        if (b%2==0 && b%3==0) {
            if (c%2==0 && c%3==0) {
                printf("%i, %i e %i são divisíveis por 2 e por 3", a, b, c);
            }
            else {
                printf("%i e %i são divisíveis por 2 e por 3", a, b);
            }
        }
        else {
            if (c%2==0 && c%3==0) {
                printf("%i e %i são divisíveis por 2 e por 3", a, c);
            }
            else {
                printf("%i é divisível por 2 e por 3", a);
            }
        }
    }
    else if (b%2==0 && b%3==0) {
        if (c%2==0 && c%3==0) {
            printf("%i e %i são divisíveis por 2 e por 3", b, c);
        }
        else {
            printf("%i é divisível por 2 e por 3", b);
        }
    }
    else {
        if (c%2==0 && c%3==0) {
            printf("%i é divisível por 2 e por 3", c);
        }
        else {
            printf("Nenhum doa números inseridos é divisível por 2 e por 3");
        }
    }
}


